snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -0,0 +1,67 @@
+version: v1
+kind: writable_storage
+name: search_issues
+
+storage:
+  key: search_issues
+  set_key: search_issues
+
+schema:
+  columns:
+    [
+      { name: organization_id, type: UInt, args: { size: 64 } },
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: group_id, type: UInt, args: { size: 64 } },
+      { name: search_title, type: String },
+      { name: fingerprint, type: Array, args: { inner_type: { type: String } } },
+      { name: occurrence_id, type: UUID },
+      { name: occurrence_type_id, type: UInt, args: { size: 8 } },
+      { name: detection_timestamp, type: DateTime },
+
+      { name: event_id, type: UUID, args: { schema_modifiers: [nullable] } },
+      { name: trace_id, type: UUID, args: { schema_modifiers: [nullable] } },
+      { name: platform, type: String, args: { schema_modifiers: [low_cardinality] } },
+      { name: environment, type: String, args: { schema_modifiers: [low_cardinality, nullable] } },
+      { name: release, type: String, args: { schema_modifiers: [low_cardinality, nullable] } },
+      { name: dist, type: String, args: { schema_modifiers: [low_cardinality, nullable] } },
+      { name: receive_timestamp, type: DateTime },
+      { name: client_timestamp, type: DateTime },
```

Comment:
After talking with @wedamija  about `client_timestamp` I think we want to keep it non-null for now since we'll always have an event here to populate the timestamp for when it was captured, regardless of whether it was captured by the SDK or some other means.